### DISCUSSION TITLE
How to update Pulsar without loose my settings and packages

### DISCUSSION THREAD

**Discussion by Raf-sns** [pulsar-edit/.github#293] [General]
Hi, 
i am using Pulsar editor on Ubuntu 24.10.
Now when i open Pulsar i have a message who encourage me to install the new version of Pulsar.
I passed much time to set Pulsar with many packages, how i can update Pulsar wihtout looze all my packages and settings?
For the settings it's not important, it's only a config file to relplace but i have many servers setted by the Package "FTP-REMOTE-EDIT" (Deprecated...) but i use it a lot and i don't want to reset all my servers settings if i install the new version of Pulsar (love it!).
Thank you for your responses and help, 
cordially - Raf

**Comment by savetheclocktower** [comment]
You asked about this already and I tried to reassure you :-)

Upgrading Pulsar will not touch your settings or packages. But if you want to make absolutely sure, you can back up the `.pulsar` directory in your home folder (`~/.pulsar`). That's where all your settings and packages are stored.

**Comment by Raf-sns** [comment]
Hi, 
Yes, of course I am worried.
When I open Pulsar I have this message:
Pulsar v1.122.0 is available. Download the latest version from the Pulsar Website or GitHub.
Dismiss this version
Dismiss until next launch
Download from GitHub

So what will happen?

Is this going to launch an `rm' command and remove Pulsar from my OS (Ubuntu/KDE Plasma)?
If so, I would definitely take everything.

Also, I have not found a list of commands to enter in my terminal in order to do this update.

I am a beginner when it comes to Linux commands.
I cannot find a procedure to follow on the Pulsar website or on the github repository to update Pulsar.

I think this procedure may seem easy to people who are used to it, but for beginners like me it remains very vague.

That is why I renewed my request for advice on the discussion space.

Thank you for your response and for your advice.
Kind regards,

Raf.

**Comment by Raf-sns** [comment]
Ok, I finally downloaded the latest .deb version of Pulsar and then used Discovery to install it and it kept all the previously installed packages.
Thanks again for your support.
Kind regards,
Raf

**Reply by confused-Techie** [reply]
No worries!
But to let you know what's going on:
Pulsar installs it's packages and keeps your settings in another directory outside of the Pulsar application itself. Meaning you can delete Pulsar entirely and your settings and installed packages are still kept safe. You can then reinstall any version of Pulsar (including a newer one) and it'll find all of your previous/existing settings and packages with no issues at all.

So in the future, feel free to redownload Pulsar from GitHub to install an update. With that said, since you are now using the `.deb` version, `pulsar-updater` will do it's best to realize that and give you the proper CLI instructions to update.

**Comment by Raf-sns** [comment]
Hi, 
thank you for your reply and your explanations, it more clear for me now.
cordially,
Raf.
